packages/gator-permissions-snap/src/profileSync/profileSync.ts

```diff
@@ -359,10 +369,12 @@ export function createProfileSyncManager(
    *
    * @param permissionContext - The context of the granted permission to update.
    * @param isRevoked - The new revocation status.
+   * @param metadata - The revocation transaction metadata.
    */
   async function updatePermissionRevocationStatus(
     permissionContext: Hex,
     isRevoked: boolean,
+    metadata?: RevocationMetadata,
```

Comment:
can we remove `isRevoked` from the signature and infer this from the existence of the `metadata`?